requirements.txt

```diff
@@ -44,3 +44,5 @@ uWSGI==2.0.10
 raven==5.2.0
 django-waffle==0.11.1
 sqlparse<0.2  #TODO: delete this when we switch to Django 1.8 and DDT 1.5
+
+pygments==2.1.3
```

Comment:
Could you add a comment so we won't forget to remove the dependency when we move PEPs to RtD?